### ISSUE TITLE
Failure plugin installation pynpp

### ISSUE THREAD

**Issue by WagnerGMD** [notepad-plus-plus/notepad-plus-plus#2147]
Hello,

I got a problem on Windows 10 (RTM 10240). I can't install the plugin **PyNPP** because it seem's to require something... Do you have an idea ? Maybe it's require a Microsoft Visual C++ Redistributable Package ? Which one ? It's something else ?

I imagine, it will be better to provide more details about the problem. Like a note where the user can download the missing file.

Sorry, I don't have much information... The downloading (2nd pictures) was aborted because the process failed.
NotePad++ version : "6.9.2"
NotePad++ Plugin Manager : "1.3.5.0"
http://nsa37.casimages.com/img/2016/08/05/16080509302894068.png
http://nsa37.casimages.com/img/2016/08/05/16080509302874342.png

The installation is **very very old** and if you can help me, it will be really appreciate.
Best regards.

**Comment by dail8859** [comment]
Does it happen for any other plugins? Do you have privileges to install plugins?

**Comment by WagnerGMD** [comment]
I don't met the trouble with another plugin (like **Combine**, **Compare** etc). Also, I only have a few plugins (<10).
On the moment, I didn't believe that I need to precise that. For me, it's normal. So, yes I have them (as I'm the administrator).

**Comment by dail8859** [comment]
It works for me using Win7. Maybe someone else can see if they have the same issue on a different OS version.

**Comment by WagnerGMD** [comment]
I don't have the issue with Windows 8.1. Nevertheless, like said I don't know (or remember) the reason.

**Comment by dail8859** [comment]
Since this appears to be a bug with the plug-in you should contact the developer if it.

**Comment by WagnerGMD** [comment]
You closed and it seem's you couldn't give even a little word (or interest) about the suggestion... No point with Notepad++ ? The Plugin Manager wasn't build by the same author, I'm wrong ? It's an important part of Notepad++, no ?
Sorry but no, it's not clear and also in that case, what's the right place (between GitHub(s), [SourceForge](https://sourceforge.net/projects/notepad-plus/), the [Community N++](https://notepad-plus-plus.org/community) , etc) ? Maybe you didn't understand ? Maybe you can't make transfert ? I don't have a **huge** knowledge about Notepad++ (GitHub etc).

From my point of view, that's weird because the installation was launched via the NotePad++ **Plugin Manager.** Apparently, it seem's to be a little lack about him. After all, he **never** purpose me a solution. Also at the first time, the problem (or origin) is absolutely not clear. Why he **never** give me even a little explanation (or a little track to follow) ? Bless my memory because no, I wasn't lucky when I searched (several times) on the internet...

Basically, I had wait a few months before I finally decide to create this report. I have been quite patient and I don't ask too much... one sign (of life etc)... If they don't seem to see the little idea which can perhaps improve (a bit) Notepad++. Then, I won't because and I don't see the point.., the report is done, none, I have done enough. 

Kind regards.

**Comment by dail8859** [comment]
I apologize if I misunderstood your post from several days ago.

Notepad++ and the Plugin Manager are created by different people.

If it is a problem with the Plugin Manager then you need to report it [here](https://github.com/bruderstein/nppPluginManager).

If it is a problem with PyNPP then you need to report it to whom ever developed it...I don't know the best place to submit it.

**Comment by Luthor2k** [comment]
This problem was fixed my my system by installing:
Microsoft Visual C++ 2010 Redistributable WithSP1 (x86)_v10.0.40219
from:
https://www.microsoft.com/en-ca/download/confirmation.aspx?id=8328
I am running:
notepad++ 7.6.6 32 bit, PyNPP 1.2, win10 pro 17763
and went to install PyNPP using the plugin manager.

Thanks Wagner for the info.
